napari/_qt/_qapp_model/qactions/_help.py

```diff
@@ -0,0 +1,21 @@
+from typing import List
+
+from app_model.types import Action
+
+from ...._app_model.constants import CommandId, MenuId
+from ...dialogs.qt_about import QtAbout
+from ...qt_main_window import Window
+
+
+def _show_about():
+    QtAbout.showAbout(Window)
+
+
+HELP_ACTIONS: List[Action] = [
+    Action(
```

Comment:
just a heads up... this will likely be a bit confusing, but if you're on mac os, you won't see this command in the help menu.  It will show up under the main application menu on the left side because of Qt's heuristics when it finds a menu item named "About <app_name>"
https://doc.qt.io/qt-6/qmenubar.html#qmenubar-as-a-global-menu-bar

that's fine, but just wanted to point out in case it was confusing

Reply:
thanks for the heads up, I was indeed confused